runtime/dancebox/tests/integration_test.rs

```diff
@@ -2410,43 +2409,6 @@ fn check_well_known_keys() {
     );
 }
 
-#[test]
-fn test_invulnerables_migration() {
```

Comment:
Are we removing this migrations?

Reply:
Yes, I removed migrations that were already applied and commented in `get_migrations` https://github.com/moondance-labs/tanssi/blob/runtime-400/runtime/dancebox/src/migrations.rs#L572 because there was a breaking change that affected one of them (don't remember exactly). Instead of trying to fix it I just removed them